bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -26,12 +26,24 @@ import pycbc.version
 
 #### DEFINITIONS AND FUNCTIONS ####
 
-def get_stat(statchoice, trigs):
+def get_stat(statchoice, trigs, threshold):
     # Initialize statclass with an empty file list. In general could feed it
     # files here for statistics which need that.
     stat_instance = sngl_statistic_dict[statchoice]([])
-    return stat_instance.single(trigs)
-
+    chunk_size = 2**23
+    stat = []
+    select = []
+    size = len(trigs['end_time'])
+    s = 0
+    while s < size:
+        e = s + chunk_size if (s + chunk_size) <= size else size
+        chunk = {k: trigs[k][s:e] for k in trigs if len(trigs[k]) == size}
```

Comment:
comment on this line, what is it supposed to do - I guess set up a 'trigs'-like object containing all possible trigger properties that could be used to calculate statistic?